04-30 00:29:23 ERROR [31m[edia._base.providers.base] Failed to parsing HD4Free: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
04-30 00:29:24 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/1337x/results/potato/?imdbid=tt2935510&search=Ad+Astra+2019&user=&passkey=xxx[0m
04-30 00:29:27 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt2935510&search=Ad+Astra+2019&user=&passkey=xxx[0m
04-30 00:29:27 ERROR [31m[hpotato.core.plugins.base] Failed opening url in TorrentPotato: http://10.0.-ag0.12:9117/api/v2.0/indexers/extratorrent/results/potato/?imdbid=tt2935510&search=Ad+Astra+2019&user=&passkey=xxx[0m
04-30 00:29:27 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/anidex/results/torznab/api?apikey=xxx&t=search&cat=&q=?imdbid=tt2935510&search=Ad+Astra+2019&user=&passkey=xxx[0m
04-30 00:29:47 ERROR [31m[edia._base.providers.base] Failed to parsing TorrentPotato: Traceback (most recent call last):
  File "/app/couchpotato/couchpotato/core/media/_base/providers/base.py", line 84, in getJsonData
    return json.loads(data)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
[0m
04-30 00:29:47 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/isohunt2/results/potato/?imdbid=tt2935510&search=Ad+Astra+2019&user=&passkey=xxx[0m
04-30 00:29:48 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/katcrs/results/potato/?imdbid=tt2935510&search=Ad+Astra+2019&user=&passkey=xxx[0m
04-30 00:29:52 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/kickasstorrent/results/potato/?imdbid=tt2935510&search=Ad+Astra+2019&user=&passkey=xxx[0m
04-30 00:29:52 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/limetorrents/results/potato/?imdbid=tt2935510&search=Ad+Astra+2019&user=&passkey=xxx[0m
04-30 00:29:55 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/magnetdl/results/potato/?imdbid=tt2935510&search=Ad+Astra+2019&user=&passkey=xxx[0m
04-30 00:29:55 INFO [0m[hpotato.core.plugins.base] Opening url: get http://10.0.0.12:9117/api/v2.0/indexers/nyaasi/results/potato/?imdbid=tt2935510&search=Ad+Astra+2019&user=&passkey=xxx[0m
04-30 00:29:56 INFO [0m[hpotato.core.plugins.base] Opening url: get https://yts.am/api/v2/list_movies.json?query_term=tt2935510&limit=10&page=1, data: [][0m
04-30 00:29:56 INFO [0m[hpotato.core.plugins.base] Opening url: get http://katcr.to/span/m-i2935510/, data: [][0m
04-30 00:29:56 INFO [0m[hpotato.core.plugins.base] Opening url: get https://pirateproxy.cat/search/%22Ad+Astra+2019%22/0/7/201, data: [][0m
04-30 00:29:56 INFO [0m[hpotato.core.plugins.base] Opening url: get https://torrentapi.org/pubapi_v2.php?get_token=get_token&app_id=couchpotato, data: [][0m
04-30 00:29:56 INFO [0m[hpotato.core.plugins.base] Opening url: get https://torrentapi.org/pubapi_v2.php?token=g6p03ekams&mode=search&search_imdb=tt2935510&min_seeders=10&min_leechers=0&ranked=1&category=movies&format=json_extended&app_id=couchpotato, data: [][0m
04-30 00:29:56 ERROR [31m[hpotato.core.plugins.base] Failed opening url in Rarbg: https://torrentapi.org/pubapi_v2.php?token=g6p03ekams&mode=search&search_imdb=tt2935510&min_seeders=10&min_leechers=0&ranked=1&category=movies&format=json_extended&app_id=couchpotato Traceback (most recent call last):
HTTPError: 429 Client Error:  for url: https://torrentapi.org/pubapi_v2.php?token=g6p03ekams&mode=search&search_imdb=tt2935510&min_seeders=10&min_leechers=0&ranked=1&category=movies&format=json_extended&app_id=couchpotato
[0m
